### PR TITLE
Store brew-build-group builds as separate results in same group

### PR DESCRIPTION
**Pull request by hluk**
Fixes storing builds in the group result - ResultsDB would store items
in the "builds" list as strings (not JSON, just stringified Python
dicts).

Signed-off-by: Lukas Holecek <hluk@email.cz>

### CODE REVIEW

**Comment by hluk** [comment]
This is same solution as for #111.

**Comment by hluk** [comment]
@thrix Would this work for you?

File: resultsdbupdater/utils.py [hunk 1]
```diff
@@ -64,6 +64,52 @@ def create_result(log, testcase, outcome, ref_url, data, groups=None, note=None)
...
+                'scratch': build['scratch'],
+
+                # optional values
+                'baseline': build.get('baseline'),
+                'dependencies': build.get('dependencies', []),
+                'source': build.get('source'),
+            }
+        except KeyError as e:
+            key_name = e.args[0]
+            raise exceptions.MissingMessageField(key_name)
```
Comment (mvalik) [inline resultsdbupdater/utils.py:106]: raise ... from e

Comment (hluk) [reply resultsdbupdater/utils.py:106]: ?

**Comment by hluk** [comment]
Closing this in favor of #119.

**Pull request closed without merging**